### ISSUE TITLE
Cannot import pyplot. NameError:  'FigureManagerWebAgg' is not defined

### ISSUE THREAD

**Issue by cknd** [matplotlib/matplotlib#9380]
### Bug report

When using the webagg backend on 2.1.0, importing pyplot fails with a NameError.

**Code for reproduction**
```python
import matplotlib
matplotlib.use('WebAgg')
from matplotlib import pyplot
```

**Actual outcome**

I get a name error. traceback excerpt:
```
.../python3.6/site-packages/matplotlib/backends/__init__.py in pylab_setup(name)
     58     # imports. 0 means only perform absolute imports.
     59     backend_mod = __import__(backend_name, globals(), locals(),
---> 60                              [backend_name], 0)
     61 
     62     # Things we pull in from all backends

.../python3.6/site-packages/matplotlib/backends/backend_webagg.py in <module>()
    325 
    326 @_Backend.export
--> 327 class _BackendWebAgg(_Backend):
    328     FigureCanvas = FigureCanvasWebAgg
    329     FigureManager = FigureManagerWebAgg

.../python3.6/site-packages/matplotlib/backends/backend_webagg.py in _BackendWebAgg()
    327 class _BackendWebAgg(_Backend):
    328     FigureCanvas = FigureCanvasWebAgg
--> 329     FigureManager = FigureManagerWebAgg
    330 
    331     @staticmethod

NameError: name 'FigureManagerWebAgg' is not defined
```

**Matplotlib version**
  * Operating system:  Mac OS 10.12.6
  * Matplotlib version: 2.1.0
  * Matplotlib backend (`print(matplotlib.get_backend())`): WebAgg
  * Python version: Python 3.6.2 
  * Jupyter version (if applicable): -
  * Other libraries: 

I installed matplotlib via pip.

**Comment by cknd** [comment]
cursory glance suggests that FigureManagerWebAgg has simply moved to `core`. this works on my machine:

(I'm on the go and can't make a proper PR right now. also, not sure this is a complete fix - when I show a figure, the browser window opens with the "WebAgg current figures" title but stays blank)


```python
# in matplotlib/backends/backend_webagg.py,  line 327
@_Backend.export
class _BackendWebAgg(_Backend):
    FigureCanvas = FigureCanvasWebAgg
    FigureManager = core.FigureManagerWebAgg   # !
```

**Comment by anntzer** [comment]
Even with the fix in #9347 this stays broken (with the symptoms described in the second message) bisecting back to  #9027.

**Comment by tacaswell** [comment]
The issue is that https://github.com/matplotlib/matplotlib/pull/6370 fixed webagg for renamed files in #5754 but the reversion did not.

**Comment by tacaswell** [comment]
PR coming shortly....got side-tracked by https://github.com/ipython/ipython/pull/10867

**Comment by anntzer** [comment]
closed by #9534, afaict
